index.html

```diff
@@ -182,16 +186,369 @@
                 <div class="featured-news-body">
                     <div class="featured-heading">FEATURED NEWS</div>
                     <div class="featured-news-sub-heading">
-                        Discover what's happening  on Azure
+                        Discover what's happening on Azure
+                    </div>
+                    <div class="feature-boxs">
+                        <div class="card">
```

Comment:
Is both card div and box div needed? If not please remove one of them and change class name to featured-card